[2024-12-13T20:27:42.251+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-13T20:27:42.475+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_score_detail manual__2024-12-13T20:27:15.522338+00:00 [queued]>
[2024-12-13T20:27:42.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_score_detail manual__2024-12-13T20:27:15.522338+00:00 [queued]>
[2024-12-13T20:27:42.487+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-13T20:27:42.502+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_data.insert_data_score_detail> on 2024-12-13 20:27:15.522338+00:00
[2024-12-13T20:27:42.511+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5438) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-13T20:27:42.513+0000] {standard_task_runner.py:63} INFO - Started process 5440 to run task
[2024-12-13T20:27:42.514+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'insert_data.insert_data_score_detail', 'manual__2024-12-13T20:27:15.522338+00:00', '--job-id', '550', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmpky0k4xna']
[2024-12-13T20:27:42.517+0000] {standard_task_runner.py:91} INFO - Job 550: Subtask insert_data.insert_data_score_detail
[2024-12-13T20:27:42.573+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_data.insert_data.insert_data_score_detail manual__2024-12-13T20:27:15.522338+00:00 [running]> on host 1fe973f28a1c
[2024-12-13T20:27:42.729+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='insert_data.insert_data_score_detail' AIRFLOW_CTX_EXECUTION_DATE='2024-12-13T20:27:15.522338+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-13T20:27:15.522338+00:00'
[2024-12-13T20:27:42.730+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-13T20:27:43.371+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-13T20:27:43.373+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-13T20:27:43.388+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=insert_data.insert_data_score_detail, run_id=manual__2024-12-13T20:27:15.522338+00:00, execution_date=20241213T202715, start_date=20241213T202742, end_date=20241213T202743
[2024-12-13T20:27:43.415+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-13T20:27:43.452+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-13T20:27:43.453+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
